test_app/tests/authentication/authenticator_plugins/test_ldap_get_or_build_user.py

```diff
@@ -0,0 +1,29 @@
+import importlib
+from unittest import mock
+from unittest.mock import MagicMock
+
+import pytest
+
+from ansible_base.authentication.authenticator_plugins import ldap
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "username",
+    [
+        ("Timmy"),
+        ("TIMMY"),
+        ("TiMmY"),
+    ],
+)
+def test_get_or_build_user(username, ldap_authenticator):
```

Comment:
Can we add a comment in this file as to why it can't be in test_ldap so that someone doesn't come along later and move it back in?